installer/datafiles/base_omsagent.data

```diff
@@ -142,6 +142,59 @@ MAINTAINER:              'Microsoft Corporation'
+
+/opt/microsoft/omsagent/tst/modules/connect/check_e2e.py;                       source/code/troubleshooter/modules/connect/check_e2e.py;                        644; root; root
+/opt/microsoft/omsagent/tst/modules/connect/check_endpts.py;                    source/code/troubleshooter/modules/connect/check_endpts.py;                     644; root; root
+/opt/microsoft/omsagent/tst/modules/connect/connect.py;                         source/code/troubleshooter/modules/connect/connect.py;                          644; root; root
+/opt/microsoft/omsagent/tst/modules/connect/__init__.py;                        source/code/troubleshooter/modules/connect/__init__.py;                         644; root; root
+
+/opt/microsoft/omsagent/tst/modules/heartbeat/check_logs.py;                    source/code/troubleshooter/modules/heartbeat/check_logs.py;                     644; root; root
+/opt/microsoft/omsagent/tst/modules/heartbeat/check_multihoming.py;             source/code/troubleshooter/modules/heartbeat/check_multihoming.py;              644; root; root
+/opt/microsoft/omsagent/tst/modules/heartbeat/heartbeat.py;                     source/code/troubleshooter/modules/heartbeat/heartbeat.py;                      644; root; root
+/opt/microsoft/omsagent/tst/modules/heartbeat/__init__.py;                      source/code/troubleshooter/modules/heartbeat/__init__.py;                       644; root; root
+
+/opt/microsoft/omsagent/tst/modules/high_cpu_mem/check_cpu.py;                  source/code/troubleshooter/modules/high_cpu_mem/check_cpu.py;                   644; root; root
+/opt/microsoft/omsagent/tst/modules/high_cpu_mem/check_logrot.py;               source/code/troubleshooter/modules/high_cpu_mem/check_logrot.py;                644; root; root
+/opt/microsoft/omsagent/tst/modules/high_cpu_mem/check_slabmem.py;              source/code/troubleshooter/modules/high_cpu_mem/check_slabmem.py;               644; root; root
+/opt/microsoft/omsagent/tst/modules/high_cpu_mem/check_space.py;                source/code/troubleshooter/modules/high_cpu_mem/check_space.py;                 644; root; root
+/opt/microsoft/omsagent/tst/modules/high_cpu_mem/high_cpu_mem.py;               source/code/troubleshooter/modules/high_cpu_mem/high_cpu_mem.py;                644; root; root
+/opt/microsoft/omsagent/tst/modules/high_cpu_mem/omiHighCPUDiagnosticsTST.sh;   source/code/troubleshooter/modules/high_cpu_mem/omiHighCPUDiagnosticsTST.sh;    644; root; root
+/opt/microsoft/omsagent/tst/modules/high_cpu_mem/__init__.py;                   source/code/troubleshooter/modules/high_cpu_mem/__init__.py;                    644; root; root
+
+/opt/microsoft/omsagent/tst/modules/syslog1/check_conf.py;                      source/code/troubleshooter/modules/syslog1/check_conf.py;                       644; root; root
```

Comment:
Minor/Nitpick: I think you can make it "TstSyslog" or similar, "syslog1" sounds like a little bit like a temporary hack rather than finalized shipped product folder :)

Reply:
Renamed it to 'syslog_tst':
https://github.com/microsoft/OMS-Agent-for-Linux/blob/09eb45de0d181e278fb5b1f83588ae0a2cc34324/installer/datafiles/base_omsagent.data#L176-L179